_maps/map_files/Deltastation/DeltaStation2.dmm

```diff
@@ -6689,6 +6683,15 @@
 	},
 /obj/effect/decal/cleanable/dirt,
 /obj/effect/turf_decal/delivery,
+/obj/machinery/computer/pod/old{
+	density = 0;
```

Comment:
Did you add this var-edit?

Reply:
the only var edits i did was pixel_x/y and id's, the rest should be copied from the icebox mass driver controller

Reply:
![image](https://user-images.githubusercontent.com/24975989/93655604-6c3a2680-fa1c-11ea-8a24-c8d42b2b4da2.png)

That's the Icebox monestary mass driver controller. The only other one is from TheDerelict.dmm ruin, but it doesn't have the density edited.
![image](https://user-images.githubusercontent.com/24975989/93655611-7fe58d00-fa1c-11ea-9977-5be1ec0bfd25.png)

Reply:
I'm gonna throw in a totally rough guess looking at those pixel values that the Icebox one is on the floor and pixel-shifted onto the wall, thus the density 0.

I'm gonna assume that the derelict one is probably placed directly on the wall, thus the small pixel_y?

Total conjecture though.

Reply:
fixed the density var edit

Reply:
Sorry if I wasn't clear here, but the `icon` and `icon_state` variables also should definitely be part of the object definition.
The `name` can be var-edited for unique items, but all of these share the same name, so they are better as an object type.

Can you move these 3 variables into a child object, since the base one uses different ones?
It would look something like this:
```cpp
/obj/machinery/computer/pod/old/mass_driver_controller
	icon = 'icons/obj/airlock_machines.dmi
	icon_state = "airlock_control_standby"
	density = FALSE
```
Then you can just substitute these instances of `/obj/machinery/computer/pod/old` for `/obj/machinery/computer/pod/old/mass_driver_controller` and removed the var-edits.

I'm assuming there's other `/obj/machinery/computer/pod/old` in maps, so it needs to be preserved, but if there are none, then directly substituting the object works as well. If so, remove the unused assets.

This can all be done via text-editing, no need to use a map editor.

Reply:
alright the only edits are the ones to id and to pixel_x/y vars. and everything is the mass_driver_controller child of pod/old